src/states/wisconsin/main.py

```diff
@@ -68,6 +68,10 @@ def query_clerk_data(pdf_data):
         result = cache_request(POST_URL, method='POST', data=post_data, wait=random.uniform(.1, .3))
         json_data = json.loads(result).get('Data') or {}
         if json_data.get('clerk'):
+          for phfax_field in ['fax', 'phone1', 'phone2']:  # remove invalid phones/faxes
```

Comment:
I'm a little confused -- why would we want to do `'\n'.join`?

Reply:
`''.join` is for the groups within the regex, `'\n'.join` is if there are multiple matches of the regex.

I don't want to return a list because it's a very uncommon case (not in current data) and would create some complex conflicts with the later pandas code. I didn't want to use `findall()[0]` to grab the first match because in some cases there are no matches. With the `'\n'.join`, anything after the first line will be truncated by `common.to_e164()`. However, I suppose it would be better to split into two lines with `findall()`, and then a check for empty list before taking the first element.